cli/objects/rule.py

```diff
@@ -274,6 +276,68 @@ def _get_jira_additional_labels(
         else:
             return None
 
+    def _get_jira_assignee(self, rule_dict: dict[Any, Any]) -> Optional[str]:
+        """
+        Determines if a Jira Assignee is defined in a rule. If it is, validate it and return the string.
+
+        :param rule_dict: A dictionary object representing a user-defined firewatch rule.
+        :return: A string of the Jira assignee to use in a firewatch rule. If one is not defined, return None
+        """
+        if "jira_assignee" in rule_dict.keys():
+            try:
+                jira_assignee = rule_dict["jira_assignee"]
+            except Exception as ex:
+                self.logger.error(ex)
+                exit(1)
+            if isinstance(jira_assignee, str):
```

Comment:
Would it be better to use a regex here?

Reply:
Looks like we had the same thought :) https://github.com/CSPI-QE/firewatch/pull/23#discussion_r1275264696